### PR TITLE
[DOCS-2031, REL-1289] Spark 4.3.0 notebook update

### PR DESCRIPTION
**Pull request by rahul-aerospike**

### CODE REVIEW

File: notebooks/spark/AerospikeSparkPython.ipynb [hunk 1]
```diff
@@ -5,7 +5,7 @@
    "metadata": {},
    "source": [
     "# Aerospike Connect for Spark Tutorial for Python\n",
-    "#### Tested with Spark connector 3.4.1, ASDB EE 6.0.0.0 EE, Java 8, Apache Spark 3.1.2, Python 3.7  and Scala 2.12.11 and [Spylon]( https://pypi.org/project/spylon-kernel/)"
+    "#### Tested with Spark connector 4.3.0, ASDB EE 6.4 EE, Java 8, Apache Spark 3.4.0, Python 3.10  and Scala 2.12.11"
```
Comment (mcoberly2) [inline notebooks/spark/AerospikeSparkPython.ipynb:8]: Scala 2.12.11 or 2.13?

File: notebooks/spark/AerospikeSparkScala.ipynb [hunk 2]
```diff
@@ -6,7 +6,7 @@
    "source": [
     "# Aerospike Spark Connector Tutorial for Scala\n",
     "\n",
-    "#### Tested with Spark connector 3.4.2, ASDB EE 6.0.0.2, Java 8, Apache Spark 3.1.2, Python 3.7 and Scala 2.12.11 and [Spylon]( https://pypi.org/project/spylon-kernel/)\n"
+    "#### Tested with Spark connector 4.3.0, ASDB EE 6.4, Java 8, Apache Spark 3.4.0, Python 3.10 and Scala 2.12.11 and [Spylon]( https://pypi.org/project/spylon-kernel/)\n"
```
Comment (mcoberly2) [inline notebooks/spark/AerospikeSparkScala.ipynb:9]: Scala 2.12.11 or 2.13?

**Review by mcoberly2 — APPROVED** [review]
FWIW LGTM

File: notebooks/spark/AerospikeSparkPython.ipynb (hunk 1 above)
Comment (rahul-aerospike) [reply notebooks/spark/AerospikeSparkPython.ipynb]: Removed Scala mention in the notebook.  I have spark 3.4.0 installed locally, which was built with Scala 2.12. I'm aware that spark version used in docker is Scala 2.13.

File: notebooks/spark/AerospikeSparkScala.ipynb (hunk 2 above)
Comment (rahul-aerospike) [reply notebooks/spark/AerospikeSparkScala.ipynb]: removed mention of scala.

**Pull request merged**